src/Allegro.DotnetSdk/Sdk/editor.globalconfig

```diff
@@ -214,6 +214,7 @@ csharp_style_expression_bodied_operators = false:none
 csharp_style_expression_bodied_properties = true:error
 csharp_style_expression_bodied_indexers = true:error
 csharp_style_expression_bodied_accessors = true:error
+csharp_style_expression_bodied_local_functions = true:error    
```

Comment:
co to zmienia? opisz prosze

Reply:
https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0061